src/docs/components/properties.md

```diff
@@ -7,219 +7,592 @@ contributors:
   - rwaskiewicz
 ---
 
-# Prop Decorator
+# Properties
 
-Props are custom attribute/properties exposed publicly on the element that developers can provide values for. Children components should not know about or reference parent components, so Props should be used to pass data down from the parent to the child. Components need to explicitly declare the Props they expect to receive using the `@Prop()` decorator. Props can be a `number`, `string`, `boolean`, or even an `Object` or `Array`. By default, when a member decorated with a `@Prop()` decorator is set, the component will efficiently rerender.
+Props are custom attributes/properties exposed publicly on an HTML element. They allow developers to pass data to a
+component to render or otherwise use.
+
+## The Prop Decorator (`@Prop()`)
+
+Props are declared on a component using Stencil's `@Prop()` decorator, like so:
 
 ```tsx
-// TodoList.tsx
-import { Prop } from '@stencil/core';
-import { MyHttpService } from '../some/local/directory/MyHttpService';
-...
+// First, we import Prop from '@stencil/core'
+import { Component, Prop, h } from '@stencil/core';
```

Comment:
~ Small detail, but other usage examples order `h` first. Likely all usage examples should be the same, either alphabetically or an explicit order.

Reply:
Good point, let me double check the Stencil CLI and match that